src/autotype/CMakeLists.txt

```diff
@@ -13,5 +13,8 @@ if(UNIX AND NOT APPLE)
 endif()
 
 if(WITH_TESTS)
-  add_subdirectory(test)
-endif()
+  # autotype non supported on Windows right now
+  if(UNIX)
```

Comment:
That file regulates only the X11 Autotype tests, that are only supported on the UNIX (not APPLE) platform right now.
This was only an "hotfix" to be able to build for Windows.

Anyway it's better to have a folder for each platform with tests inside for various feature or each feature have it's tests done on each platform (like right now)? 
Maybe you are right but we need to discuss about tests because those (x11 autotype) are inside **src/autotype/** folder and they should be in the **tests/** folder